tests/robot/_resources/test_data_sets/query/expected_results/loaded_db/B/801_get_composition_by_uid.json

```diff
@@ -2,7 +2,7 @@
   "meta": {
     "_type": "RESULTSET",
     "_created": "2019-10-18T18:12:29.282Z",
-    "_executed_aql": "SELECT c FROM COMPOSITION c [uid/value=__MODIFY_COMPOSITION_UID_1__]"
+    "_executed_aql": "SELECT c FROM COMPOSITION c [uid/value='__MODIFY_COMPOSITION_UID_1__']"
```

Comment:
@testautomation there is no need to add the quotes if the substution adds the quotes, my criteria was that, and the AQLs I created follow that pattern. If a different pattern is taken, all AQLs should be changed accordingly.

Reply:
fixed